src/components/DropDownMenu.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+import Menu from "@material-ui/core/Menu";
+import MenuItem from "@material-ui/core/MenuItem";
+import { useState } from "react";
+import { NavLink } from "react-router-dom";
+import Icon from "@material-ui/core/Icon";
+
+const DropDownMenu: React.FC = () => {
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
```

Comment:
Nie pisz EL. Napisz po prostu anchorElement, albo jeszcze lepiej podpisz co to za element np anchorSignInButton